agent.Dockerfile

```diff
@@ -5,11 +5,11 @@ FROM golang:1.20 as builder
 WORKDIR /go/src/github.com/kserve/kserve
 COPY go.mod  go.mod
 COPY go.sum  go.sum
-
 RUN go mod download
 
 COPY pkg/    pkg/
 COPY cmd/    cmd/
+RUN go mod tidy
```

Comment:
May you, please, remove changes to this file?
The `go mod tidy` is more a dev command that should not be called here when doing the build.

Reply:
oh yeah, it is a left over that I forgot to remove.
thanks